indexer-app/src/cmr/indexer/data/concepts/collection/community_usage_metrics.clj

```diff
@@ -5,9 +5,16 @@
     [cmr.indexer.data.metrics-fetcher :as metrics-fetcher]))
 
 (def not-provided-version
-  "In EMS community usage CSV, the version value when the version is unknown"
+  "DEPRECATED
+   In EMS community usage CSV, the version value when the version is unknown"
   "N/A")
 
+(defn- valid-version?
+  [parsed-version version]
+  (or (= (util/parse-version-id version) parsed-version)
```

Comment:
nit: switch the position of `parsed-version` and `(util/parse-version-id version)`.